xbmc/interfaces/json-rpc/JSONRPCUtils.h

```diff
@@ -8,10 +8,12 @@
 
 #pragma once
 
+#include "FileItem.h"
```

Comment:
Needed? Probably you can just forward declare CFileItem

Reply:
Done